spec/index.bs

```diff
@@ -11228,63 +11264,74 @@ None of the depth formats can be filtered.
             <th>{{GPUTextureSampleType}}
             <th>Valid [=image copy=] source
             <th>Valid [=image copy=] destination
+            <th>[=Texel block size=]
     </thead>
     <tr>
         <td>{{GPUTextureFormat/stencil8}}
         <td>1 &minus; 4
         <td>stencil
         <td>{{GPUTextureSampleType/"uint"}}
         <td colspan=2>&checkmark;
+        <td>1
     <tr>
         <td>{{GPUTextureFormat/depth16unorm}}
         <td>2
         <td>depth
         <td>{{GPUTextureSampleType/"depth"}}
         <td colspan=2>&checkmark;
+        <td>2
     <tr>
         <td>{{GPUTextureFormat/depth24plus}}
         <td>4
         <td>depth
         <td>{{GPUTextureSampleType/"depth"}}
         <td colspan=2>&cross;
+        <td>See prose
```

Comment:
Could we link these two instances of "see prose" to the appropriate spec text? It wasn't initially obvious to me what it was referring to.